odiglet/pkg/kube/runtime_details/inspection.go

```diff
@@ -45,6 +45,7 @@ func inspectRuntimesOfRunningPods(ctx context.Context, logger *logr.Logger, labe
 	}
 
 	if len(pods) == 0 {
+		logger.Info("No Running pods found for the workload", "name", object.GetName())
```

Comment:
Will we get this in the logs of each odiglet that does not run the workload?
I guess this can be quite a lot